site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -135,33 +135,19 @@ default, Polaris uses an in-memory metastore.
 > The default in-memory metastore is not suitable for production use, as it will lose all data
 > when the server is restarted; it is also unusable when multiple Polaris replicas are used.
 
-To use a durable metastore, you need to switch to the EclipseLink metastore, and provide your own
-`persistence.xml` file. This file contains details of the database used for metastore management and
-the connection settings. For more information, refer to the [metastore documentation]({{% ref
-"metastores" %}}).
+To use a durable metastore, you need to switch to the Relational JDBC backed metastore 
 
-Then, configure Polaris to use your metastore by setting the following properties:
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
 
-```properties
-polaris.persistence.type=eclipse-link
-polaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
-polaris.persistence.eclipselink.persistence-unit=polaris
+```
+polaris.persistence.type=relational-jdbc
```

Comment:
This is default now. There's no need for users to set it, I guess.

I suppose we may want to give an example for Data Source properties instead :thinking: ... for a local PostgreSQL server, perhaps?

Reply:
sounds fair added !